conjure-undertow-processor-example/src/main/java/com/palantir/conjure/java/undertow/example/ExampleService.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * (c) Copyright 2021 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.example;
+
+import com.google.common.collect.Iterables;
+import com.google.common.util.concurrent.ListenableFuture;
+import com.palantir.conjure.java.undertow.annotations.CollectionParamDecoder;
+import com.palantir.conjure.java.undertow.annotations.Handle;
+import com.palantir.conjure.java.undertow.annotations.HttpMethod;
+import com.palantir.conjure.java.undertow.annotations.ParamDecoder;
+import com.palantir.conjure.java.undertow.lib.BinaryResponseBody;
+import com.palantir.conjure.java.undertow.lib.RequestContext;
+import com.palantir.logsafe.Preconditions;
+import com.palantir.tokens.auth.AuthHeader;
+import io.undertow.server.HttpServerExchange;
+import java.io.Closeable;
+import java.util.Collection;
+import java.util.Optional;
+
+public interface ExampleService {
```

Comment:
What is the expectation of building a client for the generated services? Do we plan to also generate a client for this going forward? Or is the expectation that users would build/generate their own client (i.e. using [dialogue-annotations](https://github.com/palantir/dialogue/tree/3393922b5a84ccbd94e4c45498b2ec103536703d#dialogue-annotations-processor-generated-client-bindings))?

Reply:
Discussed offline. This annotation processor is primarily intended to unblock one-off consumers that are still stuck on the undertow migration, primarily due to using custom behavior that is not directly supported through generated Conjure clients.

Also generating a client based on the conjure-undertow annotations is tricky because there are subtle differences between client and server handling. Also this would come very close to re-implementing Conjure.